eng/pipelines/cadl-ci.yaml

```diff
@@ -126,7 +126,11 @@ jobs:
           publishJUnitResults: false
 
       - script: |
-          node node_modules/\@azure-tools/cadl-ranch/dist/cli/cli.js check-coverage node_modules/@azure-tools/cadl-ranch-specs/http || echo "cadl ranch tests not fully covered"
+          node node_modules/\@azure-tools/cadl-ranch/dist/cli/cli.js check-coverage node_modules/@azure-tools/cadl-ranch-specs/http
```

Comment:
do you need `--coverageFile`?

Reply:
No need, I looked at Mary and Isabella's pr, both don't define `coverageFile`, and I have tested locally, it can find the coverage file (even it's renamed) and print out the result. 
I also tried to define `--coverageFiles` locally, but the command will throw error...

Reply:
Got it work now. Able to define `--coverageFiles`, the previous error is due to something wrong in the local generated cadl-ranch-coverage-java.json.